WcaOnRails/spec/models/user_spec.rb

```diff
@@ -546,11 +546,38 @@
     end
   end
 
-  describe "#can_edit_users?" do
+  describe "#can_view_all_users?" do
     let(:competition) { FactoryGirl.create(:competition, :registration_open, :with_organizer, starts: 1.month.from_now) }
 
-    it "returns true if a user is an organizer of an upcoming comp using registration system" do
-      expect(competition.organizers.first.can_edit_users?).to eq true
+    it "returns true if the user is an organizer of an upcoming comp using registration system" do
+      organizer = competition.organizers.first
+      expect(organizer.can_view_all_users?).to eq false
+    end
+
+    it "returns true for board" do
+      board_member = FactoryGirl.create :board_member
+      expect(board_member.can_view_all_users?).to eq true
+    end
+
+    it "returns false for normal user" do
+      normal_user = FactoryGirl.create :user
+      expect(normal_user.can_view_all_users?).to eq false
+    end
+  end
+
+  describe "#can_edit_user?" do
+    let(:competition) { FactoryGirl.create(:competition, :registration_open, :with_organizer, starts: 1.month.from_now) }
+
+    it "returns true for board" do
+      board_member = FactoryGirl.create :board_member
+      user = FactoryGirl.create :user
+      expect(board_member.can_edit_user?(user)).to eq true
+    end
+
+    it "returns true for normal user" do
+      normal_user = FactoryGirl.create :user
+      user = FactoryGirl.create :user
```

Comment:
What about creating those using `let` as we do with the `competition`? This would allow you to reuse the `user` variable.
We could also move all of them (`board_member`, `normal_user`, `competition`...) up (above both `describe`s), so they would be used across the specs (if that's worth doing in your opinion).